lib/guardian_db.ex

```diff
@@ -120,4 +138,9 @@ defmodule GuardianDb do
   end
 
   def repo, do: @repo
+
+  defp storable_type?(type), do: storable_type?(type, @token_types)
```

Comment:
This could probably be simplified into:

```ex
defp storable_type?(type), do: type in @token_types
```

Reply:
The intention of these lines is to make the default behavior to store all token types if no types are specified in the config file. Making this change would make the default behavior to store no types unless explicitly added to the config file.